### DISCUSSION TITLE
Welcome to vhfinfo Discussions!

### DISCUSSION THREAD

**Discussion by htool** [htool/vhfinfo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Seanymph-RS** [comment]
Hallo Hans,

Ik vind VHF Info (de web-applicatie) een topper!
Maar ik heb wel de indruk dat voor de SignalK plugin behoefte is aan een kleine stap voor stap handleiding 'vhfinfo for Dummies´. In ieder geval voor deze dummy.
Ik ben nu al dagen achtereen aan het proberen om e.e.a. aan de praat te krijgen op mijn Raspberry Pi, maar ik kom er niet uit. Zie de bijgevoegde PDF met screenshots.

Ik zal ongetwijfeld iets verkeerd doen, maar leg e.e.a. graag voor aan mensen die er wèl verstand van hebben (ik ben zelf geen programmeur).

Voor de volledigheid (al denk ik niet dat het iets met dit probleem te maken heeft) ik gebruik geen Windows, alleen Linux/Android/Raspberry Pi OS. En ik ben dus zo brutaal om dit nu even in het Nederlands te schrijven ipv het Engels.

[Probleem VHFINFO in SignalK.pdf](https://github.com/htool/vhfinfo/files/11185623/Probleem.VHFINFO.in.SignalK.pdf)

Vriendelijke groet,
Roel (Seanymph op ZF)

**Comment by htool** [comment]
Hoi Roel,Even kort ivm met op visite:Vhfinfo heeft gps (lokatie) en heading (kompas) nodig om het searchpolygon te maken. Geen cog dus.Daarna met debug kijken  en data browser ‘vhf’ zoeken.Moet idd allemaal wat gebruiksvriendelijker. Nu bezig met wat hardware projectjes, dus nog even geen tijd.Groet,HansSent from my iPhoneOn 9 Apr 2023, at 16:12, Seanymph-RS ***@***.***> wrote:﻿
Hallo Hans,
Ik vind VHF Info (de web-applicatie) een topper!
Maar ik heb wel de indruk dat voor de SignalK plugin behoefte is aan een kleine stap voor stap handleiding 'vhfinfo for Dummies´. In ieder geval voor deze dummy.
Ik ben nu al dagen achtereen aan het proberen om e.e.a. aan de praat te krijgen op mijn Raspberry Pi, maar ik kom er niet uit. Zie de bijgevoegde PDF met screenshots.
Ik zal ongetwijfeld iets verkeerd doen, maar leg e.e.a. graag voor aan mensen die er wèl verstand van hebben (ik ben zelf geen programmeur).
Voor de volledigheid (al denk ik niet dat het iets met dit probleem te maken heeft) ik gebruik geen Windows, alleen Linux/Android/Raspberry Pi OS. En ik ben dus zo brutaal om dit nu even in het Nederlands te schrijven ipv het Engels.
Probleem VHFINFO in SignalK.pdf
Vriendelijke groet,
Roel (Seanymph op ZF)

**Comment by htool** [comment]
Hoi Roel,headingTrue is nodig. Die kun je met derived-data plugin maken indien nodig.Groet,HansSent from my iPhoneOn 9 Apr 2023, at 16:18, Hans Tool ***@***.***> wrote:﻿Hoi Roel,Even kort ivm met op visite:Vhfinfo heeft gps (lokatie) en heading (kompas) nodig om het searchpolygon te maken. Geen cog dus.Daarna met debug kijken  en data browser ‘vhf’ zoeken.Moet idd allemaal wat gebruiksvriendelijker. Nu bezig met wat hardware projectjes, dus nog even geen tijd.Groet,HansSent from my iPhoneOn 9 Apr 2023, at 16:12, Seanymph-RS ***@***.***> wrote:﻿
Hallo Hans,
Ik vind VHF Info (de web-applicatie) een topper!
Maar ik heb wel de indruk dat voor de SignalK plugin behoefte is aan een kleine stap voor stap handleiding 'vhfinfo for Dummies´. In ieder geval voor deze dummy.
Ik ben nu al dagen achtereen aan het proberen om e.e.a. aan de praat te krijgen op mijn Raspberry Pi, maar ik kom er niet uit. Zie de bijgevoegde PDF met screenshots.
Ik zal ongetwijfeld iets verkeerd doen, maar leg e.e.a. graag voor aan mensen die er wèl verstand van hebben (ik ben zelf geen programmeur).
Voor de volledigheid (al denk ik niet dat het iets met dit probleem te maken heeft) ik gebruik geen Windows, alleen Linux/Android/Raspberry Pi OS. En ik ben dus zo brutaal om dit nu even in het Nederlands te schrijven ipv het Engels.
Probleem VHFINFO in SignalK.pdf
Vriendelijke groet,
Roel (Seanymph op ZF)

**Comment by Seanymph-RS** [comment]
Thx voor de zeer snelle reactie! Ik ga verder proberen met heading ipv cog.

Vrgr, Roel



Op zo 9 apr. 2023 16:23 schreef Hans ***@***.***>:
